ddtrace/compat.py

```diff
@@ -52,6 +53,12 @@
 # DEV: `six` doesn't have `float` in `integer_types`
 numeric_types = six.integer_types + (float,)
 
+# `six.integer_types` cannot be directly used for typing
```

Comment:
Why not - can this comment be expanded a bit? Can it be now that six is unvendored?

Reply:
Added more of an explanation. If we instead did something like `Dict[_TagNameType, six.integer_types]` we get the following error:

```
ddtrace/span.py:46: error: Variable "six.integer_types" is not valid as a type  [valid-type]
ddtrace/span.py:46: note: See https://mypy.readthedocs.io/en/latest/common_issues.html#variables-vs-type-aliases
```